src/unix/notbsd/linux/other/mod.rs

```diff
@@ -436,6 +436,16 @@ extern {
     pub fn ptrace(request: ::c_uint, ...) -> ::c_long;
 }
 
+#[cfg(target_env = "gnu")]
```

Comment:
Instead of this `#[cfg]`, could you add these functions to the child modules which just aren't MUSL?